wgpu/examples/hello-copy-source/main.rs

```diff
@@ -0,0 +1,448 @@
+use bytemuck::{Pod, Zeroable};
```

Comment:
Could this example be made to use the framework to reduce the amount of code that needs to be maintained?

Reply:
If you could write a example test (following the example of cube - it's very short) this could generate the screenshot for you.

Reply:
> Could this example be made to use the framework to reduce the amount of code that needs to be maintained?

I thought about this but I need a need a async function to load image. As I know framework has sync functions. Only `setup` function is async. It will cause changing of the base methods in framework. So I didn't do that.

Reply:
Fixed.